docker/contracts/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN cargo install wasm-opt --locked
 # Clone the contracts repo
 WORKDIR /sources
 RUN git clone \
+    -b andrew/vkeys-contract \
```

Comment:
TODO: Remove this when https://github.com/renegade-fi/renegade-contracts/pull/160 is merged

Reply:
bumping this so we don't forget